tools/tsp-client/src/options.ts

```diff
@@ -0,0 +1,168 @@
+import { parseArgs } from "node:util";
+import { Logger, printUsage, printVersion } from "./log.js";
+import * as path from "node:path";
+import { knownLanguages, languageAliases } from "./languageSettings.js";
+import { doesFileExist } from "./network.js";
+import PromptSync from "prompt-sync";
```

Comment:
I saw typings for this, but not an actual dependency line?

Reply:
It's used on line 139-140 to confirm the correct output directory for the generated code.